ingest/importer/importer.py

```diff
@@ -97,9 +96,12 @@ def import_file(self, file_path, submission_url, is_update=False, project_uuid=N
             self.logger.info(f'Submission in {submission_url} is done!')
             return submission, template_mgr
 
-    def _submit_new_entities(self, entity_map, submission_url):
+    def _submit_new_entities(self, entity_map, submission_url, project_uuid):
         submission = self.submitter.add_entities(entity_map, submission_url)
         self.submitter.link_entities(entity_map, submission)
+        project = entity_map.get_project()
+        not project_uuid and project and self.submitter.link_submission_to_project(project, submission_url)
```

Comment:
Interesting. What is this doing without being assigned to a variable?

Reply:
It won't do the last statement if the first conditions/statements results to false. I think it's called shortcircuiting.

Reply:
Ok just does not make the intent clear but i get it now - thanks.